ucsc/build.sh

```diff
@@ -27,13 +28,13 @@ apt-get -qq update &&
     ORIGIN='$ORIGIN'
     export COPT ORIGIN &&
     find . -type d -maxdepth 1 -mindepth 1 -exec make -C '{}' \; &&
-    mv $HOME/bin/${arch}/* ${build}/dest/bin/ &&
-    for lib in $(ldd ${build}/dest/bin/faToTwoBit | grep -o '=> /[^ ]*' | sed 's/=> //g'); do
+    mv $HOME/bin/${arch}/* ${build}/${dest}/bin/ &&
+    for lib in $(ldd ${build}/${dest}/bin/faToTwoBit | grep -o '=> /[^ ]*' | sed 's/=> //g'); do
         case $lib in
             */libcrypto.so*|*/libgcc_s.so*|*/libmysqlclient.so*|*/libssl.so*|*/libstdc++.so*)
                 echo "including $lib"
-                cp $lib ${build}/dest/lib/
+                cp $lib ${build}/${dest}/lib/
                 ;;
         esac
     done
-    tar zcf /host/${pkg}-${version}-Linux-${arch}.tar.gz -C ${build}/dest .
\ No newline at end of file
+    tar zcf /host/${pkg}-${version}-Linux-${arch}.tar.gz -C ${build}/${dest}/../ .
```

Comment:
It would be better if `dest="${pkg}-${version}-Linux-${arch}"` at the start and here `tar zcf ${dest}.tar.gz -C ${build}/${dest}/../ .`

Otherwise I think `download_by_url` may still not work.